README.md

```diff
@@ -94,6 +94,10 @@ To publish to __https://your_username.github.io/aframe/__:
 
     npm run ghpages your_username
 
+## Contributing
+
+Get involved! Check out the [Contributing Guide](CONTRIBUTING.md) on how to get started!
```

Comment:
Oops, maybe change the last exclamation mark to a period. So we don't yell it twice :)